### PR TITLE
Report issues using the command line

### PR DESCRIPTION
**Pull request by pranavbaburaj**
- The new feature helps users to report issues with the application right from the command line
use 
`dynamic -i` to do so 

- I have also formatted all the files

### CODE REVIEW

**Review by GouravSardana — CHANGES_REQUESTED** [review]
@pranavbaburaj 

1. Paste one screenshot for reference while reporting the issue and how we can see the reported issues?
2. Why this .replit file is needed?
3. Can you check the codacy test? [1 blank line required before class docstring issues.py line 5]
4. Can we make a new folder of tests and move all the test files?

After this thing, we can set up Travis to run every time. So that nothing will break for the next time :)

**Comment by pranavbaburaj** [comment]
> @pranavbaburaj
> 
> 1. Paste one screenshot for reference while reporting the issue and how we can see the reported issues?
> 2. Why this .replit file is needed?
> 3. Can you check the codacy test? [1 blank line required before class docstring issues.py line 5]
> 4. Can we make a new folder of tests and move all the test files?
> 
> After this thing, we can set up Travis to run every time. So that nothing will break for the next time :)

I will do all of these tomorrow morning. Thank you for the suggestions 👍🏾 . I forgot to add the `.replit` file into the gitignore 🤦🏾‍♂️

File: main.py [hunk 1]
```diff
@@ -63,6 +63,12 @@
                     your Notion accout",
                     action='store_true')
 
+parser.add_argument("-i",
+										"--issue",
+                    help="\
+                    Resport an issue",
```
Comment (Siddhant-K-code) [inline main.py:69]: It should be "Report", I guess!

File: .gitignore [hunk 2]
```diff
@@ -34,4 +34,6 @@ access_token.json
 response_data.json
 
 # Selenium
-geckodriver.log
\ No newline at end of file
+geckodriver.log
+
+.replit
```
Comment (Siddhant-K-code) [inline .gitignore:39]: Add EOF

File: .replit [hunk 3]
```diff
@@ -0,0 +1,2 @@
+language = "python3"
+run = "python main.py -s"
```
Comment (Siddhant-K-code) [inline .replit:2]: add EOF

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]

**Comment by pranavbaburaj** [comment]
I think this pull request is irrelevant. I am closing this pull request. I will check out all the issues and work on some other issues

**Comment by pranavbaburaj** [comment]
I reopened this pull request because it seemed to solve #105 . But, I have also added a small command that helps us to open the browser and report an issue.
```ps1
dynamic -i
```
This command will open up the browser with the issues tab.

**Comment by pranavbaburaj** [comment]
I've checked out the issue again and I saw that @GouravSardana assigned the issue to @harshakhmk/ What do you guys suggest? Should I close this pull request ?

**Pull request closed without merging**